tensorzero-core/src/db/clickhouse/dataset_queries.rs

```diff
@@ -1100,7 +1125,8 @@ mod tests {
                 NULL as staled_at,
                 id as source_inference_id,
                 false as is_custom,
-                '' AS auxiliary
+                '' AS auxiliary,
+                snapshot_hash
```

Comment:
I think this may have been broken since when we added `snapshot_hash` but no one noticed. This PR adds some clickhouse test coverage which should help.